en/installation/debian.md

```diff
@@ -40,7 +40,7 @@ apt-get install curl
 
 ## Ruby
 
-Ruby versions packaged in official repositories are not suitable to work with consul, so we'll have to install it manually.
+Ruby versions packaged in official repositories are not suitable to work with consul (at least Debian 7 and 8), so we'll have to install it manually.
```

Comment:
IMHO this change is not necessary. The version of Ruby we use in CONSUL changes once in a while, and it'll almost never be in sync with the version offered by a distrubition.